src/js/components/bulkDownload/accounts/filters/AgencyFilter.jsx

```diff
@@ -174,112 +170,115 @@ export default class AgencyFilter extends React.Component {
 
-                <div className="download-filter__content">
-                    <div className="federal-picker">
-                        <label className={`select-label ${federalAccountDisabled}`} htmlFor="federal-select">
+                    <div className="download-filter__content">
+                        <div className="federal-picker">
+                            <label className={`select-label ${federalAccountDisabled}`} htmlFor="federal-select">
                             Federal Account
-                        </label>
-                        <div className="field-picker">
-                            <button
-                                className={`selected-button ${federalAccountDisabled}`}
-                                title={currentFederalAccountName}
-                                aria-label={currentFederalAccountName}
-                                disabled={disabled}
-                                onClick={this.toggleFederalAccountPicker} >
-                                <div className="label">
-                                    {currentFederalAccountName}
-                                </div>
-                                <div className="arrow-icon">
-                                    {federalAccountIcon}
-                                </div>
-                            </button>
+                            </label>
+                            <div className="field-picker">
+                                <button
+                                    className={`selected-button ${federalAccountDisabled}`}
+                                    title={currentFederalAccountName}
+                                    aria-label={currentFederalAccountName}
+                                    disabled={disabled}
+                                    onClick={this.toggleFederalAccountPicker} >
+                                    <div className="label">
+                                        {currentFederalAccountName}
+                                    </div>
+                                    <div className="arrow-icon">
+                                        {federalAccountIcon}
+                                    </div>
+                                </button>
 
-                            <div className={`field-list ${showFederalAccountPicker}`}>
-                                <ul>
-                                    <li className="field-item">
-                                        <button
-                                            className="item-button group-label"
-                                            title="Federal Accounts"
-                                            aria-label="Federal Accounts"
-                                            disabled>
+                                <div className={`field-list ${showFederalAccountPicker}`}>
+                                    <ul>
+                                        <li className="field-item">
```

Comment:
I don't think this first disabled option is necessary. (In the agency dropdown we use something similar to separate CFO agencies and Other Agencies)